sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryStatusAndProgressSuite.scala

```diff
@@ -137,12 +146,13 @@ object StreamingQueryStatusAndProgressSuite {
     name = "myName",
     timestamp = "2016-12-05T20:54:20.827Z",
     batchId = 2L,
-    durationMs = Map("total" -> 0L).mapValues(long2Long).asJava,
-    eventTime = Map(
-      "max" -> "2016-12-05T20:54:20.827Z",
-      "min" -> "2016-12-05T20:54:20.827Z",
-      "avg" -> "2016-12-05T20:54:20.827Z",
-      "watermark" -> "2016-12-05T20:54:20.827Z").asJava,
+    durationMs = Collections.singletonMap("total", 0L),
```

Comment:
Wait ... does this mean, that the maps we are generating in the StreamingQuery is NOT serializable?? Because we are probably doing the same conversion in ProgressReporter